templates/zerver/navbar.html

```diff
@@ -1,20 +1,30 @@
 <div id="panels">
   <div id="desktop-notifications-panel" class="alert alert-info">
```

Comment:
This element ID `desktop-notifications-panel` is no longer accurate, and some of the code elsewhere in this change gets confusing as a result -- this is now a panel which might be about either of two subjects, just one of which is desktop notifications.

Oh, I see: there's a parent, `#panels`, which is meant to be for any panel -- this one was trying to be just the desktop-notifications panel.

I think this code and layout is going to get pretty confusing to understand if we don't straighten these relationships out, so let's take care of it before merging.